tests/test_sota_checkpoints.py

```diff
@@ -507,20 +509,27 @@ def openvino_preinstall(openvino):
 
 
 @pytest.fixture(autouse=True, scope="class")
-def clean_previous_metrics_dump_dir():
-    if os.path.isdir(METRICS_DUMP_PATH):
-        subprocess.run("rm -f *.json", cwd=METRICS_DUMP_PATH, check=True, shell=True)
+def make_metrics_dump_path(metrics_dump_dir):
+    if pytest.metrics_dump_path is None:
+        data = datetime.datetime.now()
+        pytest.metrics_dump_path = PROJECT_ROOT / "test_results" / \
+            f"metrics_dump_{'_'.join([str(getattr(data, atr)) for atr in ['year', 'month', 'day', 'hour', 'minute', 'second']])}"
+    else:
+        pytest.metrics_dump_path = Path(pytest.metrics_dump_path)
```

Comment:
I think we need specify basename of metrics path like PROJECT_ROOT - 
`else:  pytest.metrics_dump_path = PROJECT_ROOT / Path(pytest.metrics_dump_path)
`
to put not full path in command line, but only the folder inside .../nncf/

Reply:
But when you specifying --sota-test-path and --sota-checkpoint-path it uses global path. As it does in our samples. It's a bit controversial for me as a test user. I think we need others opinion on this @asuhov @vshampor.

Reply:
In general, I don't think "naming output folder" is a good idea, test_sota_crckpoints in the main needed for CI, I think end users will run the models separately.

Reply:
@pfinashx I agree that test_sota_checkpoints is mainly needed for CI, but we should not forget about developers/contributors who want to validate their changes also on the checkpoints locally and control results of each run. For example, It can be helpful for bumping up the version of any third-party components. If you have not critical concerns I suggest giving developers the ability to specify any path where need to save results.